modules/deployments-comparing-deploymentconfigs.adoc

```diff
@@ -0,0 +1,112 @@
+// Module included in the following assemblies:
+//
+// * applications_and_projects/deployments/what-deployments-are.adoc
+
+[id='deployments-comparing-deploymentconfigs-{context}']
+= Comparing Deployments and DeploymentConfigs
```

Comment:
@mfojtik Who would be best to talk with about Deployments vs DeploymentConfigs? I suspect the specific features/differences lists below are out of date. We also don't have any usage docs on interacting with Deployments, just DeploymentConfigs.

Reply:
@soltysh 	@deads2k 	Thoughts on this list of supported/unsupported features? Current location for this topic now is:

https://docs.openshift.com/container-platform/4.0/applications/deployments/what-deployments-are.html#deployments-comparing-deploymentconfigs-what-deployments-are

Reply:
@adellape @tnozicka is the person to talk to